broadinstitute/single_cell_portal_core
# UX refinements for AnnData upload (SCP-5727)

#### BACKGROUND & CHANGES
This updates the 3 main steps in the AnnData upload UX.  For expression upload, an extra preamble has been added about the which data frames need to contain processed data, and where to store raw counts information:

![Screenshot 2024-08-15 at 1 51 28 PM](https://github.com/user-attachments/assets/ebf4df82-833b-4489-8c9c-44c7faa9a746)

The metadata "infographic" is updated with the new image that was added from this [PR](#2108).  Lastly, the clustering forms now will use the name of the clustering in place of the filename when the form is collapsed (this only applies once a file has been uploaded/saved):

![Screenshot 2024-08-15 at 2 00 26 PM](https://github.com/user-attachments/assets/9fc943f5-1b27-4ce7-b2dc-48173a718e41)

#### MANUAL TESTING
1. Boot as normal and load the upload wizard for an existing AnnData study
2. Confirm you see the informational text from above displayed in the `Expression matrices` tab
3. In the `Metadata` tab, confirm the image displayed talks about `adata.obs` and does not have a `NAME` column or `TYPE` row
4. In the `Clustering` tab, confirm the name of the cluster is displayed for each form, even when collapsed
